default/tests/testfiles/index.html

```diff
@@ -2,9 +2,19 @@
 <html>
     <head>
         <title>Hello world!</title>
-        <link type="text/css" rel="stylesheet" href="static/style.css">
+        <link type="text/css" rel="stylesheet" href="/static/style.css">
     </head>
     <body>
         Hello, world! =)
+        <br>
+        <div id="apiform">
+            <form method="POST" action="/userdata/">
+                <label for="apikey">API Key</label>
+                <input type="password" name="apikey" id="apikey">
+                <label for="webhook">Webhook URL</label>
+                <input type="url" name="webhook" id="webhook">
+                <input type="submit" value="Go">
+            </form>
+        </div>
     </body>
 </html>
```

Comment:
In few files there are new new lines at the end.
When we'll have CI it'll probably show warnings about it, but for now it's our problem.

Reply:
But it's for unittests and when I request page from webapp, it comes without newlines, and I check it for exact match

Reply:
Ok, I thought that there'll be newline at the ending, but probably I was wrong.